com.valvesoftware.Steam.json

```diff
@@ -140,6 +140,19 @@
                 "cp com.valvesoftware.Steam.appdata.xml /app/share/appdata",
                 "install -Dm644 ld.so.conf /app/etc/ld.so.conf"
             ]
+        },
+        {
+            "name": "lspci",
+            "buildsystem": "simple",
+            "sources": [{
+                "type": "archive",
+                "url": "https://mirrors.edge.kernel.org/pub/software/utils/pciutils/pciutils-3.6.2.tar.gz",
+                "sha256": "d5f9254f27bbda8243b345633e980144e6bd2af9c786bb8a152b904530aef599"
+            }],
+            "build-commands": [
+                "make",
+                "cp lspci /app/bin/lspci"
```

Comment:
Would this be better done by `install lspci /app/bin/lapci`? Would it make much/any difference?